src/resolver.ts

```diff
@@ -217,10 +217,16 @@ export class EthrDidResolver {
               }
               case 'svc':
                 serviceCount++
+                let endpoint
+                try {
+                  endpoint = JSON.parse(Buffer.from(currentEvent.value.slice(2), 'hex').toString())
```

Comment:
While this works, it isn't a good reference implementation.

I think a better approach is to be specific about what is being added, and specifying an encoding.
So, instead of saying `did/svc/<service name>`, one would say `did/svc/<serviceName>/j`
Or something along these lines.
We need a small adjustment to the spec to make this happen, so I'll approve this as an experimental feature for now.

Reply:
in your example the `j` stands for `json`?

Reply:
Yes, it does.
But it's just an example